File: Web-SPA/src/app/entities/seat.ts
import { Passenger } from './passenger';

export class Seat {
    passenger: Passenger;
    class: string;
    column: string;
    row: number;
    price: number;
    available: boolean;
    reserved: boolean;

    constructor(classParam: string, columnParam: string, rowParam: number, priceParam: number) {
        this.passenger = null;
        this.class = classParam;
        this.column = columnParam;
        this.row = rowParam;
        this.price = priceParam;
        this.available = true;
        this.reserved = false;
    }
}

File: Web2-Microservices/AirlineMicroservice/Models/Seat.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineMicroservice.Models
{
    public class Seat
    {
        public int SeatId { get; set; }
        public string Column { get; set; }
        public string Row { get; set; }
        public string Class { get; set; }
        public float Price { get; set; }
        public bool Available { get; set; }
        public bool Reserved { get; set; }
        public Flight Flight { get; set; }
        //public int TicketId { get; set; }
        public Ticket Ticket { get; set; }
        public Ticket2 Ticket2 { get; set; }
        //public int InvitationId { get; set; }

        public SpecialOffer SpecialOffer { get; set; }

        [Timestamp]
        public byte[] RowVersion { get; set; }
    }
}
